src/pages/_app.jsx

```diff
@@ -62,6 +63,10 @@ const WrappedApp = ({ Component, pageProps }) => {
 
   const environment = useSelector((state) => state.networkResources.environment);
 
+  useEffect(() => {
+    initTracking();
+  }, []);
```

Comment:
This shouldn't be necessary here. You can call this from within `getInitialProps` and that will guarantee that it will only be called once, when the user first loads the site. You could do something like:

```
WrappedApp.getInitialProps = async ({ Component, ctx }) => {
  const {
    store, req, query, res,
  } = ctx;
  // Do nothing if not server-side
  if (!req) { return { pageProps: {} }; }
  
  initTracking(req);

  [...]
}
```

That should solve your issues along with my other comments.